### PR TITLE
Fix diacritic in Catalan translation

### PR DESCRIPTION
**Pull request by gforcada**
I keep getting annoyed at seeing this typo in the Catalan translation.

I'm not sure what's the expected process/workflow to submit translations, I've been searching for it on jenkins.io, but the best I could get was to do:

- https://jenkins.io
- _hover_ the `Documentation` menu on the header and _click_ on __Contributor guide__
- one of the boxes says `Translate` 🎉 
- unfortunately [the link it points to](https://www.jenkins.io/doc/developer/internationalization/) is meant for developers not translators

Eventually searching for the specific typo here in this repo I found the file and now I'm making this PR 😄 

Is there a way to see how much a translation is missing, and how to edit multiple files at once, rather than having to go through all files scattered around the project? 🤔 

I've been helping translating the GNOME project to Catalan for a few years and https://l10n.gnome.org is as perfect as it can get to help translators do their job ✨ 

Is there anything similar/close to for jenkins translations? 🤔 

Thanks for the excellent software and all the steady changes that are being released!! ❤️ 

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7498"></a>

### CODE REVIEW

**Review by timja — APPROVED** [review]
I've checked in google translate and googling it, thanks for the fix

----

/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by basil — APPROVED** [review]

**Comment by gforcada** [comment]
Thanks for merging it! 🙌🏾 I will have a look at that command line tool 😄

**Pull request merged**
